website/docs/r/org_saml.html.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_saml"
+sidebar_current: "docs-vcd-resource-org-saml"
+description: |-
+  Provides a VMware Cloud Director Organization SAML resource. This can be used to create, delete, and update SAML configuration for an organization.
+---
+
+# vcd\_org\_saml
+
+Provides a VMware Cloud Director Organization SAML resource. This can be used to create, update, and delete SAML configuration for an organization.
+
+Supported in provider *v3.10+*
+
+-> **Note:** This resource requires system administrator privileges.
+
+## Example Usage with metadata file
+
+```hcl
+data "vcd_org" "my-org" {
+  name = "my-org"
+}
+
+resource "vcd_org_saml" "my-org-saml" {
+  org_id                          = data.vcd_org.my-org.id
+  enabled                         = true
+  entity_id                       = "my-entity"
+  identity_provider_metadata_file = "idp-metadata.xml"
+  email                           = "email"
+  username                        = "uname"
+  firstname                       = "fname"
+  fullname                        = "fullname"
+  surname                         = "lname"
+  role                            = "role"
+  group                           = "group"
+}
+```
+
+## Example Usage with metadata text
+
+```hcl
+data "vcd_org" "my-org" {
+  name = "my-org"
+}
+
+data "http" "example" {
+  url = "https://samltest.id/saml/idp"
+}
+
+resource "vcd_org_saml" "my-org-saml" {
+  org_id                          = data.vcd_org.my-org.id
+  enabled                         = true
+  entity_id                       = "my-entity"
+  identity_provider_metadata_text = data.http.example.response_body
+  email                           = "email"
+  username                        = "uname"
+  firstname                       = "fname"
+  fullname                        = "fullname"
+  surname                         = "lname"
+  role                            = "role"
+  group                           = "group"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org_id` - (Required) Org ID: there is only one SAML configuration available for an organization. Thus, the resource can be identified by the Org
```

Comment:
Could this explanation be rephrased? It reads strangely currently. Also, the "Org ID:" is extra/not needed.

Reply:
rephrased